src/portal/src/app/project/robot-account/add-robot/add-robot.component.ts

```diff
@@ -81,7 +81,8 @@ export class AddRobotComponent implements OnInit, OnDestroy {
                 this.isNameExisting = false;
                 this.checkNameOnGoing = true;
                 return  this.robotService.ListRobot({
-                  q: encodeURIComponent(`Level=${PermissionsKinds.PROJECT},ProjectID=${this.projectId},name=${name}`)
+                  q: encodeURIComponent(
+                      `Level=${PermissionsKinds.PROJECT},ProjectID=${this.projectId},name=${this.projectName + '+' + name}`)
```

Comment:
`Level=${PermissionsKinds.PROJECT},ProjectID=${this.projectId},name=${this.projectName + '+' + name}`?
I’m not sure if your writing is correct, I think the following writing is more in line with the norms.
`Level=${PermissionsKinds.PROJECT},ProjectID=${this.projectId},name=${this.projectName}+${name}`

Reply:
Fixed